feat: integrate keycloak in production

dymbe opened the pull request:
Integrate the deployment of Keycloak with flood-frontend

dymbe committed:
feat: integrate keycloak in production

kstigen reviewed (approved):
Add a README to this repository explaining how build and deploy works.
Also include information about what secret values are necessary and how to obtain them.

dymbe committed:
fix: remove version-tagging of keycloak-image

does not make sense to tag the keycloak image using the semantic versions of the flood-frontend image

dymbe committed:
fix: insert correct redirect URI

dymbe committed:
feat: add README
